### PR TITLE
#2 Add contact and footer

### LINKED ISSUE #2: Add contact

### PR DESCRIPTION
**Pull request by karo505**
This PR adds contact section & footer
closes #2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -201,5 +201,88 @@ <h3 class="white">Opieka nad grobami</h3>
...
+
+
+    <section id="contact">
+      <div id="contact-container">
+ 
+          <h3 class="contact-header">Kontakt</h3>
+
+
+       <div id="contact-two-columns"> 
+
```
Comment (ka1130) [inline index.html]: Maybe we could get rid of the unnecessary empty lines and fix the indents? :)

File: index.html [hunk 2]
```diff
@@ -201,5 +201,88 @@ <h3 class="white">Opieka nad grobami</h3>
...
+
+            <div id="contact-left">
```
Comment (ka1130) [inline index.html]: I can see many `id` attributes - do we need all of them? As a rule of thumb, it's always better not to overuse ids. For styling, classes and tags should be enough in most of cases. :)

File: index.html [hunk 3]
```diff
@@ -201,5 +201,88 @@ <h3 class="white">Opieka nad grobami</h3>
...
+              <h4 class="contact">ZAKŁAD POGRZEBOWY<br/> Janina Cieśla</h4>
```
Comment (ka1130) [inline index.html]: It's always better to use regular text-transform for texts. If capitalization is needed, it's better to make it in CSS via `text-transform: uppercase` :)

File: style.css [hunk 4]
```diff
@@ -343,9 +343,159 @@ section#grave-care {
...
   padding: 5rem 15rem;
   text-align: center;
 }
+
+/*---Section Documents---*/
+
+section#speaker {
+  background-color: #fff;
+  display: flex;
+  padding: 60px 0px;
```
Comment (ka1130) [inline style.css]: Maybe we could use rems here as well? :)

File: style.css [hunk 5]
```diff
@@ -343,9 +343,159 @@ section#grave-care {
...
+h3.contact-header {
+  text-align: left;
+  color: #fff;
+  font-size: 3.5rem;
+  margin-bottom:4rem;
+  font-family: var(--header-font);
+}
+
+#contact-container {
+  max-width: 1140px;
```
Comment (ka1130) [inline style.css]: ...and here too, rems instead of px? :)

File: style.css [hunk 6]
```diff
@@ -343,9 +343,159 @@ section#grave-care {
...
+    min-height: 130px;
+    display: block;
+    position: relative;
+    margin-left: auto;
+    margin-right: auto;
```
Comment (ka1130) [inline style.css]: How about `margin: auto;` ?:)

File: style.css [hunk 7]
```diff
@@ -343,9 +343,159 @@ section#grave-care {
...
+}
+
+.three-columns {
+  width:30%;
+  float: left;
+  text-align: center;
+}
+
+.three-columns:nth-child(3) {
+  padding: 0 5%;
```
Comment (ka1130) [inline style.css]: Why do we need % values here? :)

File: style.css [hunk 8]
```diff
@@ -343,9 +343,159 @@ section#grave-care {
...
+.footer-link-image {
+  width: 15.3rem;
+}
+
+
+/*---Footer---*/
+
+footer#footer {
+  background-color: #fff;
+  padding: 20px 0;
```
Comment (ka1130) [inline style.css]: How about rems? :)

**Review by ka1130 — COMMENTED** [review]
A lot of good work! And I can see new nice practices 🏅 :)
Generally, it would be great thought if code formatting and indentation was fixed.
We should also avoid styling by ids, if it's not really necessary. :)
You could consider (maybe for future projects) some styling convention, like BEM or sth alike. One nice article on BEM is here: https://blog.robertorlinski.pl/metodologia-bem/

File: index.html (hunk 1 above)
Comment (karo505) [reply index.html]: I had to install Beautify extension. Prettier doesn't work :(

Comment (ka1130) [reply index.html]: Sure, if it does the job for you then it's fine. :) Maybe indeed prettier would only work, if we had a repo with npm initialiazed, and that's not needed for now. :) The only thing that matters is that formatting is consistent, so any tool is fine. :)

File: index.html (hunk 2 above)
Comment (karo505) [reply index.html]: I removed most of ID's :)

File: index.html (hunk 3 above)
Comment (karo505) [reply index.html]: Corrected.

File: style.css (hunk 4 above)
Comment (karo505) [reply style.css]: Done :)

File: style.css (hunk 5 above)
Comment (karo505) [reply style.css]: you love rems more than me ;)

File: style.css (hunk 6 above)
Comment (karo505) [reply style.css]: Yes! :)

File: style.css (hunk 7 above)
Comment (karo505) [reply style.css]: We don't need this.

File: style.css (hunk 8 above)
Comment (karo505) [reply style.css]: ok :)

File: index.html (hunk 2 above)
Comment (karo505) [reply index.html]: And I used BEM a little.

File: style.css (hunk 5 above)
Comment (ka1130) [reply style.css]: Posssibly! :D Anyways, it's not mandatory here if you have a reason not to use them. I'd just encourage mindful usage of both units. :) Rems are better for accessibility reasons, px however play better when it comes to fine details. :)

Here are a few videos of Kevin Powell (great channel btw). You may like to have a look and decide for yourself what and when plays best for a given situation:
 - [Are you  using the right CSS units?](https://www.youtube.com/watch?v=N5wpD9Ov_To&t=12s&ab_channel=KevinPowell)
- [CSS em and rem explained](https://www.youtube.com/watch?v=_-aDOAMmDHI&ab_channel=KevinPowell)
- [Why you shouldn't set font-sizes using em](https://www.youtube.com/watch?v=pautqDqa54I&ab_channel=KevinPowell)

File: style.css (hunk 7 above)
Comment (ka1130) [reply style.css]: :) Percents are sometimes tricky, especially when working with different viewports. However in some situations they are best indeed. 

By the way, do you know of vw/vh units? Not for this case, but they are sometimes very helpful. :)
[CSS Units: vh, vw, vmin, vmax](https://www.youtube.com/watch?v=IWFqGsXxJ1E&ab_channel=KevinPowell)

File: index.html (hunk 2 above)
Comment (ka1130) [reply index.html]: Perfect <3

**Review by ka1130 — APPROVED** [review]

File: style.css (hunk 7 above)
Comment (karo505) [reply style.css]: I'm using 100vh in this project in header. I'll read about vmin and vmax and maybe use it in next project :)

File: style.css (hunk 5 above)
Comment (karo505) [reply style.css]: Thank you ❤️

File: style.css (hunk 7 above)
Comment (ka1130) [reply style.css]: Wonderful! You rock! 👯 🚀

**Pull request merged**